inst/ProjectTemplate/RShinyAppAsPkg2/R/mod_feedback.R

```diff
@@ -0,0 +1,92 @@
+#' UI for feedback module
```

Comment:
I would prefer to keep the UI and Server in a separate files.  What are the pros/cons of each approach?  I am worried the server/ui will get long in apps and starting them in the same file leads the developer to continue that way

Reply:
Yeah, I don't have a strong opinion either way. Was thinking that it will make copy/pasting files a bit simpler to have things bundled, but really shouldn't matter that much. 

Going to work on an `addModule()` function for the new template this morning and will make sure that multiple files doesn't add too much overhead.